lib/dash/content_protection.js

```diff
@@ -262,6 +262,194 @@ shaka.dash.ContentProtection.parseFromRepresentation = function(
 };
 
 
+/**
+ * Gets a Widevine license URL from a content protection element
+ * containing a custom `ms:laurl` element
+ *
+ * @param {shaka.dash.ContentProtection.Element} element
+ * @return {string}
+ */
+shaka.dash.ContentProtection.getWidevineLicenseUrl = function(element) {
+  const mslaurlNode = shaka.util.XmlUtils.findChildNS(
+    element.node, 'urn:microsoft', 'laurl');
+  if (mslaurlNode) {
+    return mslaurlNode.getAttribute('licenseUrl') || '';
+  }
+  return '';
+};
+
+
+/**
+* @typedef {{
```

Comment:
Nit: The JSDoc lines past the /** should be indented by one space, so that the * align with the first * in /**.

Reply:
Fixed